tests/metrics/github/test_security.py

```diff
@@ -62,7 +62,7 @@ def fake_vulnerabilities(client, repo):
 
 
 def test_get_repos():
-    security.query_repos = fake_repos
+    security.query.repos = fake_repos
```

Comment:
Do you know why we're not using pytest's `monkeypatch` fixture, here? It took me awhile to understand why we're accessing `repos` (or, previously, `query_repos`) through `security` and not through `query`.

Reply:
That's a good question. I tend to prefer doing it this way, but I've also had feedback from others that it's easier to understand what's happening by using `monkeypatch`. Anyway, I've taken the hint and changed it. It's more consistent with the other tests now too.